src/macros.jl

```diff
@@ -327,12 +327,14 @@ function _macro_assign_and_return(code, variable, name;
     end
 end
 
-function _check_vectorized(sense::Symbol)
+function _check_vectorized(sense::Val)
```

Comment:
Do `Val{S}) where S` to get the symbol `S` instead of doing `string(sense)` and then needed to do weird string processing